rocketpy/rocket/rocket.py

```diff
@@ -650,6 +650,7 @@ def add_motor(self, motor, position):
         self.motor_center_of_dry_mass_position = (
             self.motor.center_of_dry_mass_position * _ + self.motor_position
         )
+        self.nozzle_position = self.motor.nozzle_position * _ + self.motor_position
```

Comment:
We need to add `nozzle_position` to the documentation as well.

Reply:
Done in https://github.com/RocketPy-Team/RocketPy/pull/423/commits/9b44fbaf0204154e5aed4fac1ad7f3a2245f56b4